lumeer-core/src/main/java/io/lumeer/core/facade/UserFacade.java

```diff
@@ -50,10 +50,14 @@ public User createUser(String organizationId, User user) {
       User storedUser = userDao.getUserByEmail(user.getEmail());
 
       if (storedUser == null) {
-         return userDao.createUser(user);
+         User createdUser = userDao.createUser(user);
+         addDefaultPermissions(organizationId, createdUser.getId());
```

Comment:
Should not permissions be added before you store the entity into the database in order not to make two database requests?